src/__experimental__/components/input/input.style.js

```diff
@@ -37,6 +38,7 @@ const StyledInput = styled.input`
 
   &:invalid, &:required {
     box-shadow: none;
```

Comment:
`box-shadow` and `-webkit-box-shadow` should be the same here I think

Reply:
You are right, mistake by copy paste :3